draft-ietf-quic-multipath.md

```diff
@@ -705,7 +677,7 @@ in {{Section 8.2 of QUIC-TRANSPORT}}.
 The endhost can use all the paths in the "Active" state, provided
 that the congestion control and flow control currently allow sending
```

Comment:
You don't necessarily send stream data.
```suggestion
that the congestion control currently allow sending
```

Reply:
Yes, if you don't send stream data flow control does not apply. However, when it applies you have to consider it. I think the sentence as is, is still more correct.

Reply:
The original statement is more accurate.

Reply:
I agree that the first version is better. If a frame is not subject to flow control, the flow control always allows it.